templates/prometheus.service.j2

```diff
@@ -21,12 +21,12 @@ PrivateTmp=true
 PrivateDevices=true
 ProtectHome=true
 NoNewPrivileges=true
-{% if prometheus_systemd_version >= 231 %}
+{% if prometheus_systemd_version.stdout >= '231' %}
```

Comment:
Why not `prometheus_systemd_version.stdout | int >= 231`? We want to compare numbers not strings.